packages/gatsby/src/utils/webpack.config.js

```diff
@@ -259,6 +259,18 @@ module.exports = async (program, directory, suppliedStage) => {
       rules.images(),
       rules.media(),
       rules.miscAssets(),
+
+      // This is a hack that exports one of @reach/router internals (BaseContext)
+      // to export list. We need it to reset basepath and baseuri context after
+      // Gatsby main router changes it, to keep v2 behaviour.
+      // We will need to most likely remove this for v3.
+      {
+        test: /reach\/router\/es\/index/,
```

Comment:
This supports windows & linux. If we want to test one file we can do it with require.resolve.
```suggestion
        test: require.resolve(`@reach/router/es/index`),
```

Reply:
Oh yeah, that makes much more sense. Thanks!

Reply:
Quick question: why is this requiring `@reach/router/es/index` instead of `@reach/router/index`?